### ISSUE TITLE
Variable Expressions Bug

### ISSUE THREAD

**Issue by tao1991123** [ultraq/thymeleafjs#41]
# Code

```javascript
const { TemplateEngine, STANDARD_CONFIGURATION } = require("thymeleaf");

const templateEngine = new TemplateEngine(STANDARD_CONFIGURATION);

(async () => {
  const content1 ='<div th:attr="data-apps=${apps}"></div>';

  const html = await templateEngine.process(content1, {
    apps: [{ name: 'a'}, { name: 'b'}]
  });
  console.log("html", html);
})();
```

## Java

render result
```html
<html><head></head><body><div data-apps="[{&amp;quot;name&amp;quot;:&amp;quot;a&amp;quot;},{&amp;quot;name&amp;quot;:&amp;quot;b&amp;quot;}]"></div></body></html>
```

## node
render result

```html
<html><head></head><body><div data-apps="[object Object],[object Object]"></div></body></html>
```

**Comment by ultraq** [comment]
Looks like the expression processor needs to expand objects in the `attr` processor instead of only using `toString` on objects which results in `[object Object]`: https://github.com/ultraq/thymeleafjs/blob/997947d747b51346aab8d43a86048bf2c66748fd/source/standard/processors/AttrAttributeProcessor.js#L63

**Comment by ultraq** [comment]
Fixed in 0.20.3
